src/main/scala/com/github/codelionx/dodo/actors/Worker.scala

```diff
@@ -30,11 +30,12 @@ object Worker {
 
   case class ODFound(od: (Seq[Int], Seq[Int]))
 
+  case class GetTaskTimeout(master: ActorRef)
```

Comment:
This (and all other changes to the worker) seem to be a remainder of a discarded approach. They should be removed before merging.